src/Engine/Common/MeshChangedEventArgs.cs

```diff
@@ -2,6 +2,15 @@
 
 namespace Fusee.Engine.Common
 {
+    /// <summary>
+    /// Data for a mesh changed event
+    /// </summary>
+    public class MeshChangedEventAdditionalData
```

Comment:
@RedImp1470, @griestopf, @ASPePeX 

Please have a look. Perhaps there is a better way of doing this. I'm not very happy with this implementation. However, I somehow have to pass values (index and data) for usage within the `MeshManager` and subsequently within the `RenderContext` later on.

Reply:
There is still the problem that every call of `SetSomething(i, x)` results in an event which in turn results in the recalculation of the AABB (in the case of changing a vertex), istead of capping it to once per frame per mesh. The only way to work around that is copying the array changing the elements and copying it back (`SetSomethings(xs)`). Which is significantly faster but may still result of allot of memory allocations, especially when using our pointclouds.

I just looked over the MeshManager, and I was under the impression that it gets a call from RenderAFrame() which seems to be wrong. Since I think that most of the overhead is that every event recalculats the AABB we might want to collect all changes there and execute them once per frame.
BUT, that also means that users must be aware that Mesh data might be inconsistent until the next frame. So retrieving a AABB after Vertices are changed should either trigger the recalculation or give a warning until the dirtyflag is resolved.

I think I need to think about this a bit more.

Reply:
Here the code I tested (just drop it into Update() for the Player)

```
if (Input.Instance.KeyboardInput.IsKeyDown(KeyCodes.O))
{
    Mesh mesh = (Mesh)_scene.Children[0].Children[2].Components[1];

    for (int i = 0; i < mesh.Vertices.Length; i++)
    {
        mesh.SetVertex(i, (mesh.Vertices[i] + new float3(1, 0, 0)));
    }
}

if (Input.Instance.KeyboardInput.IsKeyDown(KeyCodes.P))
{
    Mesh mesh = (Mesh)_scene.Children[0].Children[2].Components[1];

    var verts = mesh.Vertices.ToArray();

    for (int i = 0; i < verts.Length; i++)
    {
        verts[i].x += 1;
    }

    mesh.SetVertices(verts);
}
```

Reply:
> There is still the problem that every call of `SetSomething(i, x)` results in an event which in turn results in the recalculation of the AABB (in the case of changing a vertex), instead of capping it to once per frame per mesh. [...]

Indeed. This needs to be addressed. Let's talk about this in our upcoming meeting Wednesday.